festim/exports/derived_quantities/point_value.py

```diff
@@ -2,20 +2,45 @@
 
 
 class PointValue(DerivedQuantity):
-    """DerivedQuantity relative to a point
+    """
+    Computes the value of a function at a given point
 
     Args:
-        field (str, int): the field ("solute", 0, 1, "T", "retention")
-        point (int, float, tuple, list): the point coordinates
+        field (str, int):  the field ("solute", 0, 1, "T", "retention")
+        x (int, float, tuple, list): the point coordinates
+
+    Attributes:
+        field (str, int):  the field ("solute", 0, 1, "T", "retention")
+        x (int, float, tuple, list): the point coordinates
+        show_units (bool): show the units in the title in the derived quantities
+            file
+        function (dolfin.function.function.Function): the solution function of
+            the field
+
+    Notes:
+        Units are in H/m3 for hydrogen and K for temperature
```

Comment:
```suggestion
        Units are in H/m3 for hydrogen concentration and K for temperature
```
"hydrogen" isn't a quantity